git/02.development-workflow.md

```diff
@@ -0,0 +1,115 @@
+## [Home](../README.md)
+
+## 2. Git Development Workflow
+
+This document describes the high-level process of using git as part of the
+software development process, including development commits, rewriting history,
+code reviews, squashing, rebasing, and merging.
+
+### Check project README
+
+Not all projects will conform exactly to our git best practices. Some are
+deliberate deviations; others are accidental. As a first step, check the project
+README for any mention of how to use git. Project-level instructions should
+supercede general best practices.
+
+### Create a branch
+
+Start by creating a branch from the [release branch](07.release-branching.md)
```

Comment:
I'm willing. I was considering that to be a detail covered in the release branch document and not worth mentioning here, but I suppose mentioning it here really doesn't distract too much from the high-level view. I'll add that.